index.html

```diff
@@ -0,0 +1,733 @@
+                        <li>
+                            <p>Subscription Business, +2</p>
+                        </li>
+                        <li>Index ventures, +4</li>
+                    </ul>
+
+
+                </div>
+            </div>
+
+
+            <!-- -------------------------------------------- -->
+            <div class="article-container-4">
+                <div class="article-container-1">
+
+                    <ul class="ul1">
+
+                        <li>
+                            <p>Encoded Genomics</p>
+                        </li>
+                    </ul>
+
+                    <ul class="ul2">
+                        <li class="company-logo">
+                            <img class="article-img" src="img/image.jpeg" alt="Company logo" />
+                        </li>
+                        <li>
+                        <p>$135M</p>
+                        </li>
+                        <li>
+                            <p>SERIES D</p>
+                        </li>
+                    </ul>
+
+
+
+                    <UL class="ul3">
+                        <li>
+                            <p><i class="fas fa-tag"></i>
+                                Subcription Business, +2</p>
+                        </li>
+
+                        <li>
+                            <p><i class="fas fa-tag"></i>
+                                Index ventures, +4</p>
+                        </li>
+                    </UL>
+
+                    <ul class="ul4">
+                        <li>
+                            <p>Subscription Business, +2</p>
+                        </li>
+                        <li>Index ventures, +4</li>
+                    </ul>
+
+
+                </div>
+            </div>
+
+        </div>
+
+        <!-- -----------------------categories---------------------------- -->
+
+        <section class=" more-list">
+            <div class="more-articles">
+                <p class="orange-headers">Hard Fork <i class="fas fa-chevron-right"></i></p>
+
+               <div class="hardfork-img">
+                <img class="header-img" src="img/latest-deals.png" alt="latest deals">
+               <div class="hardfork-content">
+                   <p class="thin">Most Popular</p>
+                   <p>Satoshi Nakaboto, Bitcoins hit $11,000 as it surges</p>
+                   <p class="gray">satoshi nakaboto 8hours ago</p>
+               </div>
+               </div>
+
+               
+                <div class="more-info-header">
+                    <div>
+                        <p class="orange thin">
+                            LATEST
+                        </p>
+                        <p class="orange">
+                            How your company can limit its liability for US data collection lawsuits
+                        </p>
+                    </div>
+                    <img src="img/image2.jpeg" alt="article picture">
+                </div>
```

Comment:
- [ ] In **mobile** you still need to fix the alignment
![image](https://user-images.githubusercontent.com/53256139/88811846-ab669c80-d17c-11ea-9b9c-d3ed01361200.png)